modules/importing-repositories/procedure-understanding-bulk-import-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+= Bulk Import limitations
+
+* Only the first 20 repositories (in alphabetical order) can be displayed at most on the Added Repositories page.
```

Comment:
This seems like a known issue to me for release notes instead of adding this in the feature documentation.

Reply:
I agree. This information is already in the 1.3 release notes and can be removed from here, I guess.
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/release_notes/known-issues#known-issue-rhidp-4067